src/main/java/org/apache/mesos/curator/CuratorStateStore.java

```diff
@@ -414,4 +413,33 @@ private String getTasksRootPath() {
             return tasksRootPath;
         }
     }
+
+    public boolean isSuppressed() throws StateStoreException {
+        byte[] bytes = StateStoreUtils.fetchPropertyOrEmptyArray(this, SUPPRESSED_KEY);
+        Serializer serializer = new JsonSerializer();
+
+        boolean suppressed;
+        try {
+            suppressed = serializer.deserialize(bytes, Boolean.class);
+        } catch (IOException e){
+            logger.error("Error converting property " + SUPPRESSED_KEY + " to boolean.", e);
+            throw new StateStoreException(e);
+        }
+
+        return suppressed;
+    }
+
+    public void setSuppressed(final boolean isSuppressed) {
```

Comment:
Is this used to persist `suppressed` flag, so that when a scheduler restarts, it restarts suppressed ? If yes, how  is that implemented ?